dist/assets/stylesheets/main.css

```diff
@@ -0,0 +1,141 @@
+.sidenav {
+  height: 100%; /* Full-height: remove this if you want "auto" height */
+  width: 15%; /* Set the width of the sidebar */
+  position: fixed; /* Fixed Sidebar (stay in place on scroll) */
+  z-index: 1; /* Stay on top */
+  top: 0; /* Stay at the top */
+  left: 0;
+   background-color: #111;  
+  overflow-x: hidden; /* Disable horizontal scroll */
+  padding-top: 20px;
+}
+.sidenav-content{
+  margin-top: 200px;
+}
+.dropbtn {
+  background-color: #131413;
+  color: white;
+  padding: 16px;
+  font-size: 16px;
+  border: none;
+  width: 210%;
+}
+#project-form{
+margin-top: 30px;
+}
+.projects
+{
+  cursor: pointer;
+}
+#description{
+  width: 79%;
+}
+#task-form {
+  background: gray;
+  padding: 30px 0 50px 0px;
+  border-radius: 10px;
+  /* width: 39%; */
+  margin-left: auto;
+  margin-right: auto;
+  box-shadow: 50px  rgba(0,0,0,0.2);
+  z-index: 100;
+  padding: 10px;
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 50px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
```

Comment:
## Aim to remain `DRY`
- Though having comments on what the code is doing is highly encouraged. it is also encouraged to aim at `self-documenting` code> Thereby it relieves you the burden of getting to write unnecessary comments.   
- [ ] In this instance having these comments here feel like you are repeating yourself as the comments are doing exactly what the code demonstrates. I urge we resume most of them to have a cleaner codebase. Like so 👇🏾 
```suggestion
  display: none; 
  position: fixed; 
  z-index: 1; 
  padding-top: 50px; 
  left: 0;
  top: 0;
  width: 100%; 
  height: 100%; 
  overflow: auto;
  background-color: rgb(0,0,0); 
  background-color: rgba(0,0,0,0.4); 
```